var/spack/repos/builtin/packages/curl/package.py

```diff
@@ -108,6 +111,9 @@ def configure_args(self):
             '--without-libgsasl',
             '--without-libpsl',
             '--without-zstd',
+            '--without-ca-bundle',
+            '--without-ca-path',
+            '--with-ca-fallback',
```

Comment:
Is this assuming that curl will get its certs from openssl? I'm a little concerned by this change.